tests/generate_source/build.bp

```diff
@@ -189,12 +189,38 @@ bob_binary {
         "generate_source_single_dependend",
         "generate_source_single_dependend_nested",
     ],
+    generated_headers: [
+        "generate_source_single",
+    ],
     srcs: ["main.cpp"],
 }
 
+bob_generate_source {
+    name: "gen_foo_sources",
```

Comment:
Pref: Maybe you could call this something like "gen_sources_and_headers" to better explain what this is testing?